docs/Architecture Decision Record/016-dependabot-dependency-management.md

```diff
@@ -35,6 +34,15 @@ To maintain our gitflow:
 
 * (minor) The GitHub README status badges currently don't work for the GitHub Native Dependabot: [as noted in this open dependabot issue](https://github.com/dependabot/dependabot-core/issues/1912)
 
+## Development Dependencies
+This project defines a number of "Development Dependencies" which are not installed in production-esque environments (ie. deployed in Cloud.gov) but are instead only used for supporting operations during local development and automated unit testing in CI/CD.
+
+These dependencies will still be kept up to date by Dependabot but needn't follow the same review flow as updates to packages we actively use within the codebase outside of development/testing. In those cases, as long as the tests all pass on the branch opened to update the dev dependency it may be merged directly into `raft-tdp-main`.
+
+Dev dependencies are listed in two locations in this project:
```

Comment:
@abottoms-coder @jtwillis92 just curious, would it be useful to configure auto-merge for these dev dependencies after all checks pass? not sure if its feasible, as ive seen conflicting info around the web re: efficacy of [this](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/upgrading-from-dependabotcom-to-github-native-dependabot#differences-between-dependabot-preview-and-github-native-dependabot)

Reply:
I could see it being useful, looks like there is an [open source GitHub Action](https://github.com/ahmadnassri/action-dependabot-auto-merge) that would be able to do what we want here for only development dependencies.

Reply:
In the interim, can we switch the major-minor-patch filter for the spammy awscli in the dependabot? I want to avoid specifically awscli updating every two days so perhaps we ignore patch updates and only updates on the minor version updates. I'll make a follow-on issue for this and link it here.

Reply:
just documenting for the record: discussed this with Andrew on Friday, and he noted that there arent too many dev dependencies to manage manually, and awscli is the only that is updated pretty often, hence the above suggestion. So for now, let's try this manual merge process for a bit to see how it goes. When PRs are opened on the HHS side, we'll need the links to the merged PR included in the summary, to facilitate a check to ensure that only devdeps are being merged into raft-tdp-main w/o qasp review.  cc: @lfrohlich